checkov/common/util/tqdm_utils.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import os
+import sys
+
+from colorama import Fore, Back
+from tqdm import tqdm  # type: ignore
+
+DEFAULT_BAR_FORMAT = '{l_bar}%s{bar:20}%s|[{n_fmt}/{total_fmt}]{postfix}' % (Fore.WHITE, Fore.RESET)
+SLOW_RUNNER_BAR_FORMAT = '{l_bar}%s{bar:20}%s|[{n_fmt}/{total_fmt}] %s[Slow Runner Warning]%s{postfix}' %\
+                         (Fore.LIGHTBLACK_EX, Fore.RESET, Back.YELLOW, Back.RESET)
```

Comment:
@nimrodkor I can't use f-strings here since the tqdm variables are also wrapped with `{...}` and these variables are not locally present (l_bar, bar, etc)

Reply:
This should work
```suggestion
DEFAULT_BAR_FORMAT = f'{{l_bar}}{Fore.WHITE}{{bar:20}}{Fore.RESET}|[{n_fmt}/{total_fmt}]{postfix}' % (Fore.WHITE, Fore.RESET)
SLOW_RUNNER_BAR_FORMAT = '{{l_bar}}{Fore.LIGHTBLACK_EX}{{bar:20}}{Fore.RESET}|[{{n_fmt}}/{{total_fmt}}] {Back.YELLOW}[Slow Runner Warning]{Back.RESET}{{postfix}}'
```

Reply:
Worked, cool trick :)